ocp_utilities/infra.py

```diff
@@ -81,6 +85,73 @@ def assert_pods_failed_or_pending(pods):
         )
 
 
+def assert_nodes_in_healthy_condition(
+    nodes,
+    bypass_check_if_kubelet_ready=True,
+    healthy_node_condition_type=None,
+):
+    """
+    Validates nodes are in a healthy condition.
+    Nodes Ready condition is True and none of the following node conditions is True:
+        - DiskPressure
+        - MemoryPressure
+        - PIDPressure
+        - NetworkUnavailable
+
+    Args:
+         nodes(list): List of Node objects
+
+         bypass_check_if_kubelet_ready:
```

Comment:
Please fix the docstring
`arg (type): description`

Reply:
done